openeo/extra/stac_job_db.py

```diff
@@ -0,0 +1,303 @@
+import concurrent
```

Comment:
I think we should start making `job_manager` a package instead of module, so that files are organised like

```
- openeo
    - extra
        - job_management
            - __init__.py
            - stac_job_db.py
```

Reply:
With a811bffb99e0c7071fdd68e1350505d2f419d95f  I made `openeo.extra.job_management` a package now, so you can move this new module to `openeo.extra.job_management.stac_job_db` now

Reply:
I also merge master now in this feature branch to resolve all conflicts.

so make sure to pull first before continuing on this feature branch